index.bs

```diff
@@ -422,6 +417,21 @@ To perform a <dfn abstract-op>cross-origin resource policy check</dfn> given a [
         extensions, and I think it'll be more difficult to ship them after inverting the
         error-handling behavior.
 
+Integration with Service Worker {#integration-sw}
+-------------------------------------------------
+
+In https://w3c.github.io/ServiceWorker/#dom-fetchevent-respondwith, replace 10.1 with the following
+item.
+
+1. If |response| is not a `Response` object, or the result of performing a
+[$cross-origin resource policy check$] with _event_'s request's associated request and _response_'s
+associated response is `blocked`, then set the respond-with-error flag.
+
+Also add the following note.
+
+The [$cross-origin resource policy check$] needs to be performed _after_ the relevant service
```

Comment:
I don't think the text works anymore, as you're putting the check in ServiceWorker itself, while I was imagining that this would be a post-request check, similar to step 12 of https://fetch.spec.whatwg.org/#main-fetch. Now it's happening as part of the `.respondWith()` algorithm itself. Perhaps something along the lines of:

"""
The Cross-Origin Resource Policy check performed here ensures that a Service Worker cannot respond to a client that requires CORP with an opaque response that doesn't assert CORP.
"""

Reply:
Thanks, fixed.